modules/@ergonode/ui/src/components/Paragraph/Paragraph.vue

```diff
@@ -21,10 +21,11 @@ export default {
         if (!isEmpty(this.$slots)) {
             paragraphContent = this.title.split(' ').map((word) => {
                 // pattern to search `[[slot_name]]`
-                const regex = /\[\[([a-zA-Z1-9_]+)\]\]/g;
-                const key = word.replace(regex, '$1');
+                const regex = /\[\[([0-9a-zA-Z]+)\]\]/g;
+
+                if (regex.test(word)) {
+                    const key = word.replace(/\[|\]/g, '');
 
-                if (regex.test(word) && this.$slots[key]) {
```

Comment:
Cuz it will fail, there is no need for checking if slot for given key exists - your intention is to create slot not to check if exists - it's been created dynamically